webpack.config.shared.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   config: {
     entry: {
       root:  ["@babel/polyfill", "./static/js/entry/root"],
+      root:  ["core-js/stable", "regenerator-runtime/runtime", "./static/js/entry/root"],
```

Comment:
Should the line above be removed or should this be under a different key?

Reply:
lol :woman_facepalming: